integration_tests/src/main/python/hash_aggregate_test.py

```diff
@@ -603,52 +603,27 @@ def test_hash_pivot_groupby_duplicates_fallback(data_gen):
     ('b', value_gen)] for value_gen in _repeat_agg_column_for_collect_list_op]
 
 _repeat_agg_column_for_collect_set_op = [
-    RepeatSeqGen(all_basic_struct_gen_no_floats, length=15),
+    RepeatSeqGen(all_basic_struct_gen, length=15),
     RepeatSeqGen(StructGen([
-        ['c0', all_basic_struct_gen_no_floats], ['c1', int_gen]]), length=15)]
-
-struct_array_gen_no_nans = StructGen([['child'+str(ind), sub_gen] for ind, sub_gen in enumerate(single_level_array_gens_no_nan)])
-
-struct_array_gen_no_floats = StructGen([['child'+str(ind), sub_gen] for ind, sub_gen in enumerate(single_level_array_gens_no_floats)])
+        ['c0', all_basic_struct_gen], ['c1', int_gen]]), length=15)]
 
 # data generating for collect_set based-nested Struct[Array] types
 _repeat_agg_column_for_collect_set_op_nested = [
-    RepeatSeqGen(struct_array_gen_no_floats, length=15),
+    RepeatSeqGen(struct_array_gen_no_nans, length=15),
```

Comment:
Why are we using a no NaNs form here?  I thought this PR is supposed to be fixing the NaN handling.